Sources/TuistLoader/ProjectDEscriptionHelpers/ProjectDescriptionHelpersBuilder.swift

```diff
@@ -27,66 +33,130 @@ final class ProjectDescriptionHelpersBuilder: ProjectDescriptionHelpersBuilding
     /// Project description helpers hasher.
     let projectDescriptionHelpersHasher: ProjectDescriptionHelpersHashing
 
+    /// The name of the default project description helpers module
+    static let defaultHelpersName = "ProjectDescriptionHelpers"
+
     /// Initializes the builder with its attributes.
     /// - Parameters:
     ///   - projectDescriptionHelpersHasher: Project description helpers hasher.
     ///   - cacheDirectory: Path to the cache directory.
     ///   - helpersDirectoryLocating: Instance to locate the helpers directory.
-    init(projectDescriptionHelpersHasher: ProjectDescriptionHelpersHashing = ProjectDescriptionHelpersHasher(),
-         cacheDirectory: AbsolutePath = Environment.shared.projectDescriptionHelpersCacheDirectory,
-         helpersDirectoryLocator: HelpersDirectoryLocating = HelpersDirectoryLocator())
-    {
+    init(
+        projectDescriptionHelpersHasher: ProjectDescriptionHelpersHashing = ProjectDescriptionHelpersHasher(),
+        cacheDirectory: AbsolutePath = Environment.shared.projectDescriptionHelpersCacheDirectory,
+        helpersDirectoryLocator: HelpersDirectoryLocating = HelpersDirectoryLocator()
+    ) {
         self.projectDescriptionHelpersHasher = projectDescriptionHelpersHasher
         self.cacheDirectory = cacheDirectory
         self.helpersDirectoryLocator = helpersDirectoryLocator
     }
 
-    func build(at: AbsolutePath, projectDescriptionSearchPaths: ProjectDescriptionSearchPaths) throws -> AbsolutePath? {
-        guard let helpersDirectory = helpersDirectoryLocator.locate(at: at) else { return nil }
-        if let cachedPath = builtHelpers[helpersDirectory] { return cachedPath }
+    func build(
```

Comment:
maybe it'd be good to test the building logic for plugins, too? Either in a new unit test or in `ProjectDescriptionHelpersBuilderIntegrationTests`, what do you think?

Reply:
Added!